proto/dydxprotocol/accountplus/models.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
For my understanding, are these essentially copies from Osmosi protos? Did you make any changes?

Reply:
yup - mostly just copies from the Osmosis protos

for params, omitted [maximum_unauthenticated_gas](https://github.com/osmosis-labs/osmosis/blob/362a01b08cd27d8e22e6c733a6f2c8c75b1d34df/proto/osmosis/smartaccount/v1beta1/params.proto#L10-L14) and [circuit_breaker_controllers](https://github.com/osmosis-labs/osmosis/blob/362a01b08cd27d8e22e6c733a6f2c8c75b1d34df/proto/osmosis/smartaccount/v1beta1/params.proto#L22-L25)